qa/suites/fs/upgrade/mds_upgrade_sequence/tasks/3-upgrade-mgr-staggered.yaml

```diff
@@ -0,0 +1,19 @@
+teuthology:
+  premerge: |
+            if not yaml.teuthology.variables.fail_fs then reject() end
+upgrade-tasks:
+  sequential:
+    - cephadm.shell:
+        env: [sha1]
+        host.a:
+          - ceph config set mon mon_warn_on_insecure_global_id_reclaim false --force
+          - ceph config set mon mon_warn_on_insecure_global_id_reclaim_allowed false --force
+          - ceph config set global log_to_journald false --force
+          - ceph orch upgrade start --image quay.ceph.io/ceph-ci/ceph:$sha1 --daemon-types mgr
+          - while ceph orch upgrade status | jq '.in_progress' | grep true && ! ceph orch upgrade status | jq '.message' | grep Error ; do ceph orch ps ; ceph versions ; ceph orch upgrade status ; sleep 30 ; done
+          - ceph versions | jq -e '.mgr | length == 1'
+          - ceph versions | jq -e '.mgr | keys' | grep $sha1
+          - ceph versions | jq -e '.overall | length == 2'
+          - ceph orch upgrade check quay.ceph.io/ceph-ci/ceph:$sha1 | jq -e '.up_to_date | length == 2'
+          - ceph orch ps
+          - ceph config set mgr mgr/orchestrator/fail_fs true
```

Comment:
Okay. This is annoying to work around but let's do this:

````
4-config-upgrade/
  +
  fail_fs.yaml
````
````
teuthology:
  premerge: |
          if not yaml.teuthology.variables.fail_fs then reject() end
upgrade-tasks:
  sequential:
    - cephadm.shell:
        env: [sha1]
        host.a:
         - ceph config set mgr mgr/orchestrator/fail_fs true
````

Reply:
4-config-upgrade will be part of `1-volume` dir, right?

Reply:
no, it needs to happen after 3-upgrade-mgr-staggered.yaml

Reply:
@batrick I dont understand this
```
4-config-upgrade/
  +
  fail_fs.yaml

Reply:
That's the new layout I'm proposing.

Reply:
Yeah I pushed this change a few hours and also have put some tests on teuthology.